force-app/main/default/classes/OpportunityTriggerHandler.cls

```diff
@@ -0,0 +1,12 @@
+public with sharing class OpportunityTriggerHandler {
+    public static void doNotDeleteBankingWonOpps(List<Opportunity> listOfOpps) {
+        List<Opportunity> oppsToSave = [SELECT Name, Id, Account.Id, Account.Industry, StageName FROM Opportunity WHERE Id IN :listOfOpps];
+        For (Opportunity opp : oppsToSave){
+            If (opp.Account.Industry == 'Banking'){
+                If (opp.StageName == 'Closed Won'){
```

Comment:
`for` and `if` are not capitalized. also, we should combine these if statements so the code is not deeply nested and harder to read:
```suggestion
        for (Opportunity opp : oppsToSave) {
            if (opp.Account.Industry == 'Banking' && opp.StageName == 'Closed Won') {
```